flexget/plugins/cli/series.py

```diff
@@ -299,6 +328,16 @@ def register_parser_arguments():
     begin_parser.add_argument('episode_id', metavar='<episode ID>',
                               help='Episode ID to start getting the series from (e.g. S02E01, 2013-12-11, or 9, '
                                    'depending on how the series is numbered)')
+
+    addshow_parser = subparsers.add_parser('add', parents=[series_parser],
+                                           help='Add episode(s) and season(s) to series history')
+    addshow_parser.add_argument('entity_id', nargs='+', default=None, metavar='<entity_id>',
+                                help='Episode or season entity ID(s) to add')
+    addshow_parser.add_argument('--quality', default=None, metavar='<quality>',
+                                help='Quality string to be stored for all entity ID(s)')
+    addshow_parser.add_argument('--not-downloaded', action='store_true', dest='not_downloaded',
```

Comment:
Maybe if you're using it in a perverse way to track what you want to get by inputting undownloaded stuff? I know that sounds weird. I just figured since it was capable of displaying undownloaded releases, we should allow them to be added, but I don't have a solid use case in mind.